pkg/cli/client/client.go

```diff
@@ -55,6 +55,7 @@ func GetClientFromConfig(
 func GetClient(
 	serverAddress string,
 	credential string,
+	proxyCredential string,
 	insecureTLS bool,
```

Comment:
As the number of optional details here grows, it's probably better that we think about introducing a `ClientOptions` struct. The signature would change to:

```
func GetClient(
	serverAddress string,
	opts *ClientOptions,
) svcv1alpha1connect.KargoServiceClient
```

It should be tolerant of nil options.